hooks/useProposals.ts

```diff
@@ -191,8 +210,15 @@ export const useProposals = () => {
         description: proposal.description,
         isBridged: Boolean(l2Proposal),
         status: {
-          l1: proposal.isCancelled ? 'cancelled' : 'open',
-          l2: l2Proposal?.isCancelled ? 'cancelled' : 'open',
+          l1: proposal.isCancelled
+            ? 'cancelled'
+            : l1Block && l1Block > BigInt(proposal.endBlock)
+            ? 'closed'
+            : 'open',
```

Comment:
Not a huge fan of nested ternaries. One of these could be abstracted to a variable

Reply:
i'd be down. the proposal statuses need a total revamp, so i think i'll defer this change until then. tracking in #122